lib/ims/lti/models/messages/message.rb

```diff
@@ -1,6 +1,8 @@
 module IMS::LTI::Models::Messages
   class Message
 
+    attr_reader :simple_oauth_header
```

Comment:
It's too make it easier to debug OAuth signing issues. The current case is to compare base strings between TC and TP. I have been monkey patching, but decided to do something better.